core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy

```diff
@@ -9,11 +9,9 @@ l.layout {
     l.header(title:"Jenkins")
     l.main_panel {
         h1 {
-            l.icon(class: 'icon-error icon-xlg')
```

Comment:
I don't feel strongly about this, but perhaps the existing `close-circle`, `close`, or `indeterminate` icons would work for this. Or we could add `alert`, `alert-circle`, `bug`, `nuclear`, or `warning` from upstream Ionicons. Can be tested with

```java
diff --git a/core/src/main/java/jenkins/model/Jenkins.java b/core/src/main/java/jenkins/model/Jenkins.java
index 66b7a9e6ab..bbdd39a359 100644
--- a/core/src/main/java/jenkins/model/Jenkins.java
+++ b/core/src/main/java/jenkins/model/Jenkins.java
@@ -4338,9 +4338,9 @@ public class Jenkins extends AbstractCIBase implements DirectlyModifiableTopLeve
                 try (ACLContext ctx = ACL.as2(ACL.SYSTEM2)) {
                     reload();
                     getLifecycle().onReady();
+                    new JenkinsReloadFailed(new Throwable()).publish(servletContext, root);
```

and reloading Jenkins.

Reply:
To be fair, this is a left over from the header icon cleanup, I didn't want to create a dedicated PR form when I noticed it.

I can do so if you want.

Reply:
I am not familiar with the header icon cleanup. If this is consistent with some previously accepted PR that you can link to, that is fine with me.

Reply:
This took place across a series of PRs modernizing different components, for example https://github.com/jenkinsci/jenkins/pull/5925/files#diff-58a21354a8d8b1ac4db72b5a141485cd14f888acd5f32e5148db76779c753bafL33
The goal was to remove icons next to headers, as it is the current state with the majority of pages.

Reply:
OK, would have been nice if this was done consistently throughout the repository in a single commit, but as long as this is consistent with previous changes I am fine with it.